### ISSUE TITLE
Theme Header & Footer not shown on job posting page

### ISSUE THREAD

**Issue by marcoluig** [glebpineparks/job-postings#56]
I use the default theme "Twenty Twenty-Three" and Wordpress v6.2.
On the detail page, the header and footer from the template are not displayed.

**Comment by stefanpuls** [comment]
Yes. Same problem here. It looks like the plugin doesn't work with „Full Site Editing” (Template Editing).

Twenty Twenty-Three v1.1 | WordPress v6.2.  | Plugin v2.6.1

**Comment by mma-jb-2** [comment]
Same Problem here!
Twenty Twenty-Three v1.1 | WordPress v6.2. | Plugin v2.6.1
